credentials.json

```diff
@@ -0,0 +1,5 @@
+{"installed":{"client_id":"181032288929-dsna30mqeooco6fluvdennrtansnq268.apps.googleusercontent.com",
+  "project_id":"quickstart-1601290410416","auth_uri":"https://accounts.google.com/o/oauth2/auth",
+  "token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs",
```

Comment:
could some of these values be moved to the ENV file?

Reply:
Possible, however this seems to be googles standard. Some information is extracted, I am just not sure of all the locations this file is used